src/d3a/d3a_core/sim_results/energy_trade_profile.py

```diff
@@ -120,3 +126,22 @@ def add_sold_bought_lists(res_dict):
                         list(res_dict[area_name][ks][node]["accumulated"].keys())
                     res_dict[area_name][ks + "_lists"][node]["energy"] = \
                         list(res_dict[area_name][ks][node]["accumulated"].values())
+
+    @staticmethod
+    def convert_timestamp_strings_to_timestamps(profile):
+        outdict = {}
+        for k in profile.keys():
+            outdict[k] = {}
+            for sold_bought in ("sold_energy", "bought_energy"):
+                if sold_bought not in profile[k]:
+                    continue
+                outdict[k][sold_bought] = {}
+                for dev in profile[k][sold_bought].keys():
+                    outdict[k][sold_bought][dev] = {}
+                    for target in profile[k][sold_bought][dev].keys():
+                        outdict[k][sold_bought][dev][target] = {}
+                        for timestamp_str in profile[k][sold_bought][dev][target].keys():
+                            timestamp_object = str_to_pendulum_datetime(timestamp_str)
```

Comment:
Correct me if Im wrong, from the description of method name, I thought it will convert timestamp strings to timestamps  whereas it actually converted to datetime object.

Reply:
Well, datetime and timestamp is the same, isn't it? I could change it to `convert_timestamp_strings_to_timestamp_objects` or `convert_timestamp_strings_to_datetimes` but this is not easier to read IMO.

Reply:
> Well, datetime and timestamp is the same, isn't it? 

 Okay, Im not sure actually, the default method of pendulum `timestamp()` returns a UNIX_timestamp, so I related the same and expected here the `unix timestamp` too

Reply:
Ahhhh I see now, that the source of the confusion was the UNIX timestamp. In this case I agree with you, renaming this to use datetime. Thanks!